scripts-vagrant_provision/provision.yml

```diff
@@ -0,0 +1,175 @@
+# In order, vagrantfile goes:
+# DOS2UNIX (should probably stay)
+# install and run it
+# PREREQUISITES
+# installs curl, python stuff, python via pyenv, node via nvm, python global packages
+# creates folders
+# INSTALL DOCKER/COMPOSE/PIN
+# OTHER_UTILS
+# make jq net-tools vim sqlite3
+# BASHRC
+# ORCHESTRATE
+# set .env, start, fix frontend
+
+---
+
+- name: Prerequisites - folders
+  hosts: all
+  become: yes
+  tags: ['folders']
+
+  tasks:
+
+    - name: Set permissions of /opt
+      file:
+        dest: "/opt"
+        owner: vagrant
+        group: vagrant
+        mode: g=u
+        recurse: yes
+
+    - name: Create directories and set permissions
+      file:
+        path: "{{ item }}"
+        state: directory
+        owner: vagrant
+        group: vagrant
+        recurse: yes
+      loop:
+        - /opt/myaegee
+        - /opt/backups
+
+    - name: Check current Nodejs version
```

Comment:
Why would we check the version here if we're checking in the next step that it's not installed?

Reply:
if it's our version, we don't uninstall it
if it's another version, we completely uninstall (and let the role take care of it)
in fairness, if the role is done properly, they should already check if there's a node and uninstall it, so maybe i will remove this

Reply:
after checking the role, it's better if it stays